### PR TITLE
Efiling LookupDemo Service

### PR DESCRIPTION
**Pull request by KFloodBCGov**
# Description

This PR includes the following proposed change(s):

DemoLookupServiceImpl for the demo application
.gitignore for tests folder

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Local build/unit tests

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/libs/efiling-lookup-client/src/main/java/ca/bc/gov/open/jag/efilinglookupclient/ServiceFees.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.joda.time.*;
+
+import java.math.BigDecimal;
+
+
+public class ServiceFees {
+
+    @SuppressWarnings("squid:S00107")
```
Comment (akroon3r) [inline src/backend/libs/efiling-lookup-client/src/main/java/ca/bc/gov/open/jag/efilinglookupclient/ServiceFees.java:12]: do we need a comment here about why we're supressing this?

Comment (KFloodBCGov) [reply src/backend/libs/efiling-lookup-client/src/main/java/ca/bc/gov/open/jag/efilinglookupclient/ServiceFees.java:12]: Done!

Comment (akroon3r) [reply src/backend/libs/efiling-lookup-client/src/main/java/ca/bc/gov/open/jag/efilinglookupclient/ServiceFees.java:13]: Thank you :)

**Comment by akroon3r** [comment]
59% diff compared to 45%. Thank you @KFloodBCGov

**Pull request merged**